packages/e2e/test/web-extension/dapp-sdk/features/setExpiredValidityIntervalTx.ts

```diff
@@ -0,0 +1,36 @@
+import { EMPTY, catchError, take, tap } from 'rxjs';
+import { inspectAndSignTx } from '../utils';
+
+import { Cardano } from '@cardano-sdk/core';
+import type { ObservableWallet } from '@cardano-sdk/wallet';
+
+export const setExpiredValidityIntervalTx = async ({
```

Comment:
I think that's a more appropriate name, as it builds a new tx, rather than setting it on an existing one
```suggestion
export const buildExpiredValidityIntervalTx = async ({
```

Reply:
it's addressed here: 6169d4881e48a53c5c5065cdc8a2d1b804d7902b